site/src/examples/number-input/FixedDecimalScale.tsx

```diff
@@ -0,0 +1,27 @@
+import { FormField, FormFieldLabel } from "@salt-ds/core";
+import { NumberInput } from "@salt-ds/lab";
+
+export const FixedDecimalScale = () => {
+  return (
+    <FormField style={{ width: "256px" }}>
+      <FormFieldLabel>Number input with fixed decimal scale</FormFieldLabel>
+      <NumberInput
+        defaultValue={100.25}
+        decimalScale={4}
+        onNumberChange={(_event, newValue) =>
+          console.log(`Number changed to ${newValue}`)
+        }
+        format={(value: string) => {
```

Comment:
```suggestion
        format={(value) => {
```

I'm guessing this isn't needed?